Bottom: 5a85dbcc4aeda09b546416c49294ff9a3ce63fdd
Top:    8efe426ae2ffd9b8634376c9996e17db17730b88
Author: Ian Kent <raven@themaw.net>
Date:   2013-03-21 12:13:57 +0800

Refresh of autofs-5.0.5-add-master-read-wait-option.patch

---

diff --git a/lib/defaults.c b/lib/defaults.c
index 688461a..ed521b7 100644
--- a/lib/defaults.c
+++ b/lib/defaults.c
@@ -593,7 +593,7 @@ int defaults_get_master_wait(void)
 	if (wait < 0)
 		wait = DEFAULT_MASTER_WAIT;
 
-	return (unsigned int) wait;
+	return (int) wait;
 }
 
 unsigned int defaults_get_negative_timeout(void)
